versioned_docs/version-8.1/components/connectors/out-of-the-box-connectors/aws-sns-inbound.md

```diff
@@ -0,0 +1,156 @@
+3. Fill in all required properties.
+4. Complete your BPMN diagram.
+5. Deploy the diagram to activate the webhook.
+6. Navigate to the **Webhooks** tab in the properties panel to see the webhook URL.
+
+## Make your AWS SNS Inbound Connector for receiving notifications executable
+
+![Start event SNS connector](../img/connectors-aws-sns-inbound-start-filled.png)
+
+![Intermediate catch event SNS connector](../img/connectors-aws-sns-inbound-intermediate-filled.png)
+
+1. In the **Subscription Configuration** section, configure the **Subscription ID**. By default, **Subscription ID** is pre-filled with a random value. This value will be a part of the topic subscription URL.
+2. Set the **Allow to receive messages from topic(s)** value to **any** if the process may be triggered by any topic, or **Specific topic(s)** if you wish to allow-list only certain topics to start a new BPMN process.
+3. If you have chosen the **Specific topic(s)**, you have to list comma-separated topics in the field **Topic ARN(s)** as well. In that case, the **AWS SNS Inbound Connector** will auto-approve each qualified subscription request.
+4. In the section **Activation**, configure **Condition** when the AWS SNS topic can trigger a new BPMN process. The following example will trigger a new BPMN process for every notification with a subject _Start BPMN_: `=(request.body.Subject = "Start BPMN")`.
+5. In the section **Variable mapping** fill the field **Result variable** to store the response in a process variable. For example, `myResultVariable`.
+6. In the section **Variable expression** fill the field to map specific fields from the response into process variables using [FEEL](/components/modeler/feel/what-is-feel.md).
+   The following example will extract both message and subject from AWS SNS message: `={message: request.body.Message, subject: request.body.Subject}`.
+
+When using the **AWS SNS Inbound Connector** with an **Intermediate Catch Event**, fill in the **Correlation key (process)** and **Correlation key (payload)**.
+
+- **Correlation key (process)** is a FEEL expression that defines the correlation key for the subscription. This corresponds to the **Correlation key** property of a regular **Message Intermediate Catch Event**.
+- **Correlation key (payload)** is a FEEL expression used to extract the correlation key from the incoming message. This expression is evaluated in the Connector Runtime and the result is used to correlate the message.
+
+For example, given that your correlation key is defined with `myCorrelationKey` process variable, and the request body contains `"MessageAttributes": {"attrName1" : {"Type":"String","Value":"attrVal"}}`, your correlation key settings will look like this:
+
+- **Correlation key (process)**: `=myCorrelationKey`
+- **Correlation key (payload)**: `=request.body.MessageAttributes.attrName1.Value`
+
+Learn more about correlation keys in the [messages guide](../../../concepts/messages).
```

Comment:
is the link broken?

Reply:
Working for me. Opens 
<img width="736" alt="Screenshot 2023-06-07 at 16 08 33" src="https://github.com/camunda/camunda-platform-docs/assets/108870003/f71bdec8-1263-4360-b430-dc66b8345f99">

We also have a scanner that checks for broken links, I believe.

Reply:
The build would fail if this link didn't work. I usually correct these as I see them to include the top folder/directory so:

`/components/concepts/messages`

But both ways should work!